calculators/helpers/PetNetworthHelper.js

```diff
@@ -59,6 +61,14 @@ class PetNetworthHelper {
         return SOULBOUND_PETS.includes(this.petData.type);
     }
 
+    /**
+     * Checks if the pet is cosmetic
+     * @returns {boolean} Whether the pet is cosmetic
+     */
+    isCosmetic() {
+        return !!this.skin;
+    }
```

Comment:
Well, the whole pet shouldn't be excluded... Just the skin part of the pet should be excluded.